src/vs/workbench/services/editor/browser/editorService.ts

```diff
@@ -514,6 +515,26 @@ export class EditorService extends Disposable implements EditorServiceImpl {
 		return overrides;
 	}
 
+	private registerDefaultOverride(): void {
+		this._register(this.editorOverrideService.registerContributionPoint(
```

Comment:
It is not entirely clear to me why this is needed, would be great to clarify in a comment

Reply:
Comment added.